### ISSUE TITLE
Stopping and resuming network does not bring up all containers successfully

### ISSUE THREAD

**Issue by KevinSmall** [ConsenSys/quorum-dev-quickstart#55]
The quickstart demo network works correctly when started using `run.sh`. However, after executing `stop.sh` and `resume.sh`, not all containers get brought back up correctly. 

This is running with WSL2 (Ubuntu 20.04.2 LTS), Windows 10 (version 2004 build 19041), Docker for Windows 20.10.5.

## To Replicate
1. Execute `npx quorum-dev-quickstart`. During setup, answers are: use goquorum, orchestrate no, private yes, ELK no.
1. Execute `./run.sh`. All nodes start successfully.
1. cakeshop shows node status running, peers = 8, blocks are produced.
1. Execute `docker-compose ps` it shows all containers as `Up` or `Up (healthy)`.
1. Execute `./stop.sh`. All nodes stop successfully.
1. Execute `docker-compose ps` shows all containers as `Exit..`
1. Execute `./resume.sh.` Output should be similar to this:
```
*************************************
Quorum Dev Quickstart
*************************************
Resuming network...
----------------------------------
Starting validator1     ... done
Starting validator2     ... done
Starting validator3     ... done
Starting validator4     ... done
Starting rpcnode        ... done
Starting ethsignerProxy ... done
Starting member1tessera ... done
Starting member1quorum  ... done
Starting member2tessera ... done
Starting member2quorum  ... done
Starting member3tessera ... done
Starting member3quorum  ... done
Starting explorer       ... done
Starting cakeshop       ... done
```
The above all shows green, however all is not well. cakeshop shows 5 nodes (not 8). Blocks are produced ok, though.

8. Execute `docker-compose ps`, shows tessera doesn't start, and quorum nodes are unhealthy:
```
$ docker-compose ps
                Name                              Command                   State                                                     Ports
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
quorum-test-network_cakeshop_1         /bin/sh -c rm -rf /data/ca ...   Up (healthy)     8080/tcp, 8102/tcp, 0.0.0.0:8999->8999/tcp
quorum-test-network_ethsignerProxy_1   /opt/ethsigner/bin/ethsign ...   Up               0.0.0.0:18545->8545/tcp
quorum-test-network_explorer_1         /docker-entrypoint.sh ngin ...   Up               0.0.0.0:25000->80/tcp
quorum-test-network_member1quorum_1    /bin/sh -c                       Up (unhealthy)   0.0.0.0:60191->30303/tcp, 30303/udp, 0.0.0.0:20000->8545/tcp, 0.0.0.0:20001->8546/tcp, 8547/tcp
quorum-test-network_member1tessera_1   /bin/sh -c mkdir -p /var/l ...   Exit 0
quorum-test-network_member2quorum_1    /bin/sh -c                       Up (unhealthy)   0.0.0.0:60193->30303/tcp, 30303/udp, 0.0.0.0:20002->8545/tcp, 0.0.0.0:20003->8546/tcp, 8547/tcp
quorum-test-network_member2tessera_1   /bin/sh -c mkdir -p /var/l ...   Exit 0
quorum-test-network_member3quorum_1    /bin/sh -c                       Up (unhealthy)   0.0.0.0:60194->30303/tcp, 30303/udp, 0.0.0.0:20004->8545/tcp, 0.0.0.0:20005->8546/tcp, 8547/tcp
quorum-test-network_member3tessera_1   /bin/sh -c mkdir -p /var/l ...   Exit 0
quorum-test-network_rpcnode_1          /bin/sh -c mkdir -p /var/l ...   Up (healthy)     0.0.0.0:60190->30303/tcp, 30303/udp, 0.0.0.0:8545->8545/tcp, 0.0.0.0:8546->8546/tcp, 8547/tcp
quorum-test-network_validator1_1       /bin/sh -c mkdir -p /var/l ...   Up (healthy)     0.0.0.0:60187->30303/tcp, 30303/udp, 0.0.0.0:21001->8545/tcp, 8546/tcp, 8547/tcp
quorum-test-network_validator2_1       /bin/sh -c mkdir -p /var/l ...   Up (healthy)     0.0.0.0:60188->30303/tcp, 30303/udp, 0.0.0.0:21002->8545/tcp, 8546/tcp, 8547/tcp
quorum-test-network_validator3_1       /bin/sh -c mkdir -p /var/l ...   Up (healthy)     0.0.0.0:60185->30303/tcp, 30303/udp, 0.0.0.0:21003->8545/tcp, 8546/tcp, 8547/tcp
quorum-test-network_validator4_1       /bin/sh -c mkdir -p /var/l ...   Up (healthy)     0.0.0.0:60183->30303/tcp, 30303/udp, 0.0.0.0:22004->8545/tcp, 8546/tcp, 8547/tcp
```
Trying `./stop.sh` and `./resume.sh` sometimes leaves quorum nodes hanging in status restarting.
Executing `./stop.sh`, `./remove.sh` and `./run.sh` brings up all nodes successfully again.

## Possible Workaround
The fault may be related to the fact that `resume.sh` calls each container's entrypoint again. I managed to get the network to be brought back up correctly by editing `docker-compose.yml` and replacing all `mkdir` that don't have the `-p` option, with `mkdir -p`. I am not convinced this is the real fault, though, because the mkdir lines without the -p should just be executed and ignored if the directory already exists. Also, this workaround sometimes needed me to call `./stop.sh` and `./resume.sh` more than once before network came back correctly. So root cause is likely elsewhere, but the above may give a clue.

**Comment by joshuafernandes** [comment]
Hi @KevinSmall thanks for reporting this issue. It should be fixed by #59 

Cheers

**Comment by KevinSmall** [comment]
Thanks Josh, I can confirm this works well now.
